clients/go/test/util/public_key_server.go

```diff
@@ -19,29 +19,47 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
+	"io"
 	"log"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"strings"
 
 	"github.com/lestrrat-go/jwx/v2/jwk"
 )
 
-// Matching private key here: https://github.com/abcxyz/lumberjack/blob/92782c326681157221df37e0897ba234c5a22240/integration/testrunner/grpcrunner/grpc.go#L60
-const publicKeyString = `
------BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhBWj8vw5LkPRWbCr45k0cOarIcWg
-ApM03mSYF911de5q1wGOL7R9N8pC7jo2xbS+i1wGsMiz+AWnhhZIQcNTKg==
------END PUBLIC KEY-----
-`
-
 // StartLocalPublicKeyServer parse pre-made key and set up a server to host it in JWKS format.
 // This is intended to stand in for the JVS in the integration tests.
+type publicKeyJSONData struct {
+	Encoded string
+}
+
+func loadJSON() (*publicKeyJSONData, error) {
+	var data *publicKeyJSONData
+	jsonFile, err := os.Open("/etc/lumberjack/public_key.json")
+	if err != nil {
+		return nil, fmt.Errorf("failed to open file: %w", err)
+	}
+	byteValue, _ := io.ReadAll(jsonFile)
```

Comment:
nit: prefer short local names 
```suggestion
	b, _ := io.ReadAll(jsonFile)
```